File: Transport/JS/Transport/view/RefuellingGroup.js
T.view.RefuellingGroup = Ext.extend(Kdn.view.BaseGrid, {
    modelName: 'RefuellingGroup',
    constructor: function(cfg) {
        cfg = cfg || {};
        Ext.apply(cfg, {
            
            colModel: new Ext.grid.ColumnModel({
               defaults:{filter:{}},            
                columns: [
                    {
                        dataIndex: 'RefuellingGroupId',
                        header: 'Код',
                        width: 130,
                        editor: { xtype: 'kdn.editor.id' }
                    },
                    {
                        dataIndex: 'RefuellingGroupName',
                        header: 'Наименование',
                        width: 300,
                        editor: { xtype: 'kdn.editor.textfield' }
                    },
                    {
                        dataIndex: 'groupAccCode',
                        header: 'Код',
                        width: 100
                    },
                    {
                        dataIndex: 'oilReplacingReport',
                        header: 'Отчёт по маслам',
                        width: 100
                    },
                    {
                        dataIndex: 'debit',
                        header: 'Код затрат',
                        width: 100
                    },
                    {
                        dataIndex: 'oilDebit',
                        header: 'Код затрат масел',
                        width: 100
                    }
                ]
            })
        });

        T.view.RefuellingGroup.superclass.constructor.call(this, cfg);

    }
});

Ext.reg('view.refuellinggroup', T.view.RefuellingGroup);

File: Transport/Models/refuelling/RefuellingGroup.cs
using Castle.ActiveRecord;
using Kdn.Ext.Attributes;

namespace Transport.Models {
   [Model, ActiveRecord]
   public class RefuellingGroup : ActiveRecordBase<RefuellingGroup> {
      
      [IdProperty,PrimaryKey(Generator=PrimaryKeyType.Assigned)]
      public int RefuellingGroupId { get; set; }
      [Property(Length=30)]
      public string RefuellingGroupName { get; set; }
      [Property(Length = 20)]
      public string groupAccCode { get; set; }
      [Property]
      public bool? oilReplacingReport { get; set; }
      [Property(Length = 10)]
      public string debit { get; set; }
      [Property(Length = 10)]
      public string oilDebit { get; set; }
   }
}
